src/components/services/ConfigService.js

```diff
@@ -225,8 +225,8 @@
             state: 'help.variants.summary'
           },
           {
-            heading: 'CIViC Actionability Score',
-            state: 'help.variants.actionabilityscore'
+            heading: 'Evidence Score',
```

Comment:
Should this be `Variant Evidence Score`?

Reply:
With the full name, the tab row wraps to two lines - given the context it seems clear that the Evidence Score relates to variants.